### ISSUE TITLE
CSIP21 dependent on whether files exist or not?

### ISSUE THREAD

**Issue by PhillipAasvangTommerholt** [DILCISBoard/E-ARK-CSIP#701]
When creating test corpuses we have created two packages:

[One with files in the metadata/descriptive-folder ](https://github.com/DILCISBoard/eark-ip-test-corpus/blob/integration/corpora/csip/metadata/dmdsec/CSIP21/IP_18000_CSIP21_1.zip)
[One without files in the metadata/descriptive-folder.](https://github.com/DILCISBoard/eark-ip-test-corpus/blob/integration/corpora/csip/metadata/dmdsec/CSIP21/IP_18000_CSIP21_2.zip)
Both packages have no mdRefs in dmdSec in METS.xml.

I have trouble in understanding the CSIP21 requirement. Is it a SHOULD-requirement because there should be descriptive metadata in the package in general or is it a SHOULD even in both scenarios where files exist or not? 

I would intuitively say that a mdRef-pointer MUST exist if there are files in the metadata/descriptive-folder and a SHOULD if there are no files.

**Comment by prettybits** [comment]
> I would intuitively say that a mdRef-pointer MUST exist if there are files in the metadata/descriptive-folder and a SHOULD if there are no files.

I share this intuition, which the description of the test case also [expresses](https://github.com/DILCISBoard/eark-ip-test-corpus/blob/19cdc2b4e32699d92d6a1d1e330eece2814c7790/corpora/csip/metadata/dmdsec/CSIP21/testCase.xml#LL17C5-L17C69). As to the general point _why_ it is SHOULD I think this together with CSIP17 follows from CSIPSTR7. CSIP17 is SHOULD because it is allowed for there to be no descriptive metadata ("If descriptive metadata are available") and CSIP21 is SHOULD because CSIPSTR7 itself is, allowing in principle for metadata to not be placed in the strongly recommended place in the folder structure (or use an `mdWrap` element inline instead?).

**Comment by karinbredenberg** [comment]
Yes, if there are descriptive metadata the first rule to follow is CSIP17. Then its one dmdSec per descriptive metadata so therefore CSIP21 have 0..1 and a should.
